ST_structs.h

```diff
@@ -147,6 +147,8 @@ struct resourcegroup_st {
         estabs,         /* total indivs in group established during iter */
         killyr,         /* kill the group in this year; if 0, don't kill, but see killfreq */
         yrs_neg_pr,     /* counter for consecutive years low resources */
+       	wildfire, 		/*number of wildfires during all iterations*/
+       	prescribedfire, /*number of prescribed fires during all iterations*/
```

Comment:
`/* number of prescribed fires during all iterations in this year */`

Reply:
Commit 983faa7 resolves this.